install_pim/manual/common_install_ce_ee.rst.inc

```diff
@@ -0,0 +1,75 @@
+Initializing Akeneo
+-------------------
+
+Configure the PIM via the ``.env`` file:
+
+.. code-block:: yaml
+    :linenos:
+
+    APP_ENV=prod
+    APP_DATABASE_HOST=localhost
+    APP_DATABASE_PORT=null
+    APP_DATABASE_NAME=akeneo_pim
+    APP_DATABASE_USER=akeneo_pim
+    APP_DATABASE_PASSWORD=akeneo_pim
+    APP_INDEX_HOSTS='elastic:changeme@elasticsearch:9200'
+
+
+Launching the PIM in dev mode
+-----------------------------
+
+.. note::
+
+   All `make` commands must be run from the PIM root directory, either created by the archive or from the composer create project above.
+
+.. code-block:: bash
+
+	$ NO_DOCKER=true make dev
```

Comment:
```suggestion
	$ NO_DOCKER=true make pim-dev
```

Reply:
no, this is _this_ Makefile https://github.com/akeneo/pim-enterprise-dev/blob/master/std-build/Makefile

Reply:
:+1: